### DISCUSSION TITLE
@umijs/max配置子应用后跳转到子应用后再点击父应用菜单跳转，路由路径不正确，求解！！！

### DISCUSSION THREAD

**Discussion by SSSonnettt** [umijs/umi#8439] [Q&A]
1、通过手脚架插件项目；
2、注册并引用子应用；
![image](https://user-images.githubusercontent.com/34835334/177597481-9a4db490-c47c-4bbf-989c-4de596e94506.png)
![1657124470](https://user-images.githubusercontent.com/34835334/177598633-63f4a389-5c0f-4b7d-bf2b-bf619ed57e64.png)
3、跳转子应用（vue），路由路径正常
![1657124572](https://user-images.githubusercontent.com/34835334/177598729-b9227c0a-2761-4e87-943b-72a51c9912c1.png)
4、但是这时候点击父应用上面的菜单时，路由路径把子应用的path加上了
![image](https://user-images.githubusercontent.com/34835334/177599432-faa87ee6-bcde-4228-a4df-fce9e04a754f.png)

求解应该怎么做才能正常跳转？？？

**Comment by fz6m** [comment]
给个复现 demo 看看。
